oshi-core/src/main/java/oshi/software/common/AbstractOSProcess.java

```diff
@@ -72,6 +75,11 @@ public double getProcessCpuLoadBetweenTicks(OSProcess priorSnapshot) {
         return getProcessCpuLoadCumulative();
     }
 
+    @Override
+    public List<OSThread> getThreadDetails() {
+        return Collections.emptyList();
```

Comment:
Just a reminder, this was a placeholder while developing all the implementations. Once you do Solaris this can/should be removed.

Reply:
yes sure.